spec/views/new_gestionnaire/avis/instruction.html.haml_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe 'new_gestionnaire/avis/instruction.html.haml', type: :view do
+  let(:gestionnaire) { create(:gestionnaire) }
+  let(:dossier) { create(:dossier) }
+  let(:avis) { create(:avis, claimant: gestionnaire, confidentiel: confidentiel) }
+
+  before do
+    assign(:avis, avis)
+    @dossier = create(:dossier, state: "accepte")
+    allow(view).to receive(:current_gestionnaire).and_return(gestionnaire)
```

Comment:
Ça ça me semble pas top car ça veut dire qu'on est dans le cas d'une auto-demande d'avis, ça serait mieux avec un autre gesionnaire. On peut donc renommer le tout premier en `claimant_gestionnaire` et celui-là en `expert_gestionnaire` pour clarifier.

Reply:
Dans Factory:avis, les `claimant_gestionnaire` et `expert_gestionnaire` ont été initialiser. la gestionnaire ici est `claimant`.

Reply:
Là on veut tester l'affichage pour l'expert, pas le claimant il me semble

Reply:
OK, on peut l'utiliser `avis.gestionnaire` pour `:current_gestionnaire`
`allow(view).to receive(:current_gestionnaire).and_return(avis.gestionnaire)`
Et je vais supprimer `let(:gestionnaire) { create(:gestionnaire) }` et `claimant: gestionnaire,` ici